op-bindings/Makefile

```diff
@@ -80,3 +80,10 @@ clean:
 
 test:
 	go test ./...
+
+test-bindgen:
+	ETHERSCAN_APIKEY_ETH=$(ETHERSCAN_APIKEY_ETH) \
+	ETHERSCAN_APIKEY_OP=$(ETHERSCAN_APIKEY_OP) \
+	RPC_URL_ETH=$(RPC_URL_ETH) \
+	RPC_URL_OP=$(RPC_URL_OP) \
```

Comment:
You shouldn't need to add this. The env vars should already be available to the subprocess.

Reply:
Oh right, thank you 🙂